pages/_formats/genome-coordinates.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: "Interbase Genome Coordinates"
+title: "Genome Coordinates"
 layout: default
 date: 2019-01-08
-author: 
+author:
   - "@jmarshall"
   - "@andrewyatz"
-  - "@mbaudis"
```

Comment:
Permalink needed (though now w/o the _interbase); otherwise ant re-structuring of the site will break links.
Authorship ... also including @reece, since part of the discussion.

Reply:
Permalink added — it was not clear whether a permalink convention had been settled on!